### ISSUE TITLE
pylint swears at the correct use of gettext

### ISSUE THREAD

**Issue by kingco6ra** [PyCQA/pylint#7885]
### Bug description

```python
"""non-empty docstring"""
from gettext import gettext as _
print(_('qwe %r') % 'asd')
```

### Command used

```shell
pylint foobar.py
```

### Pylint output

```shell
************* Module foobar
foobar.py:3:20: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)

------------------------------------------------------------------
Your code has been rated at 5.00/10 (previous run: 5.00/10, +0.00)
```

### Expected behavior

pylint will consider this a valid case

### Pylint version

```shell
pylint 2.13.7
astroid 2.11.6
Python 3.10.8 (main, Nov 14 2022, 00:00:00) [GCC 12.2.1 20220819 (Red Hat 12.2.1-2)]
```

### OS / Environment

Fedora 36

**Comment by clavedeluna** [comment]
I'm not able to reproduce the `consider-using-f-string` msg. Could you please upgrade to a newer pylint version?
